########################################
# Messages
########################################

uint8 AorM
uint8 Gear
float32 KPH
float32 Cal_KPH
float32 Enc_Cal_KPH
float32 Deg
uint8 Brake
int32 Encoder